printer-creality-cr10v3-bltouch.cfg

```diff
@@ -0,0 +1,229 @@
+#Stock Creality CR-10 V3 with BL-Touch and runout filament sensor
+[stepper_x]
+step_pin: PF0 #ar54
+dir_pin: PF1 #ar55
```

Comment:
Does this printer have a 8bit board? If not, remove all the #arXX comments.

Reply:
Yes, it does have an 8-bit mainboard.